migrations/1623975493095-add-crypto-units-to-config.js

```diff
@@ -0,0 +1,22 @@
+const { saveConfig } = require('../lib/new-settings-loader')
+
+exports.up = function (next) {
+  const cryptoUnits = {
+    wallets_BTC_cryptoUnits: 'mili',
+    wallets_ETH_cryptoUnits: 'mili',
+    wallets_LTC_cryptoUnits: 'mili',
+    wallets_ZEC_cryptoUnits: 'mili',
+    wallets_BCH_cryptoUnits: 'mili'
```

Comment:
We only want to add this if the wallet is configured. Also dash is missing

Reply:
Hmm, I thought we wanted to force this default value for all wallets. otherwise, we must force a default value throughout the codebase so we don't display undefined somewhere. Is it really worth it?